src/meson.build

```diff
@@ -106,8 +106,9 @@ executable('chr', main,
   link_with: editor_lib,
   dependencies : [qt5_dep, tuiwidgets_dep, posixsignalmanager_dep, syntax_dep])
 
-executable('tests', tests,
+tests_bin = executable('tests', tests,
   qt5.preprocess(moc_headers: tests_headers, moc_sources: tests,
   include_directories: include_directories('.')),
   link_with: editor_lib,
   dependencies : [qt5_dep, tuiwidgets_dep, posixsignalmanager_dep, syntax_dep, catch2_dep])
+test('tests', tests_bin, workdir: meson.current_build_dir())
```

Comment:
It seems layout=flat is deprecated by meson, so the workdir should be ok even if it doesn't support flat layout.

Reply:
I think we should run tests in verbose mode, so information is output directly in build logs instead of having to refer to testlog.txt which is often hard in disto or CI build.

Verbose mode is not supported on all supported meson versions though, so something like this would be needed:
https://github.com/tuiwidgets/tuiwidgets/commit/813534d09bad83224d154d53f4f76c72dbfae29a